src/data/transforms/minmax.rs

```diff
@@ -82,9 +82,12 @@ impl<T: Float> MinMaxScaler<T> {
 }
 
 impl<T: Float> Transformer<Matrix<T>> for MinMaxScaler<T> {
-    fn transform(&mut self, mut inputs: Matrix<T>) -> Result<Matrix<T>, Error> {
+
+    fn fit(&mut self, inputs: &Matrix<T>) -> Result<(), Error> {
         let features = inputs.cols();
 
+        // ToDo: can use min, max
+        // https://github.com/AtheMathmo/rulinalg/pull/115
```

Comment:
Just to note that this implementation may actually be more efficient as we handle it on one pass of the rows.

Certainly worth checking though (besides - we will have some other breaking changes by the time this PR lands).